src/segments/segment_linebreak.rs

```diff
@@ -0,0 +1,11 @@
+use std::borrow::Cow;
+use crate::{Powerline, Segment, Shell};
+
+pub fn segment_linebreak(p: &mut Powerline) {
+    let (bg, fg) = (0, 0);
+    p.segments.push(match p.shell {
+        Shell::Bare => Segment::new(bg, fg, { Cow::from(String::from("\n")) }).with_no_space_after(),
```

Comment:
Why is this different from the rest?

Reply:
This was probably a mistake. It's addressed now. :)
Yes, this could replace `--newline`. To be fair, `--newline` wasn't working with me at all.